website/static/js/projectSettings.js

```diff
@@ -68,11 +69,17 @@ var ProjectSettings = oop.extend(
                 self.changeMessage(language.updateSuccessMessage, 'text-success');
                 return;
             }
-            var requestPayload = self.serialize();
-            var request = $osf.ajaxJSON('patch',
-                self.updateUrl,
-                { data: requestPayload,
-                isCors: true });
+            var requestPayload = JSON.stringify(self.serialize());
+            var request = $3.ajax({
+                    url: self.updateUrl,
+                    type: 'PATCH',
+                    dataType: 'json',
+                    contentType: 'application/json',
```

Comment:
If this is to a v2 api url, it should probably be 'application/vnd.api+json' otherwise this is good.